pyproject.toml

```diff
@@ -11,6 +17,9 @@ dependencies = [
     "bitstruct>=8.12",
 ]
 
+[project.urls]
+"Source" = "https://github.com/google/avatar"
```

Comment:
nit: Do we need the quotes here ?

Reply:
Looking at the doc it seems that we don't https://flit.pypa.io/en/latest/pyproject_toml.html#urls-table

```suggestion
Source = "https://github.com/google/avatar"
```

Reply:
I just followed the tutorial and the quotes were here
https://packaging.python.org/en/latest/tutorials/packaging-projects/#configuring-metadata

I'll remove them, looks cleaner without